src/test/java/com/neuvector/ScannerTest.java

```diff
@@ -9,34 +9,46 @@
 
 import org.junit.Test;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.attribute.FileOwnerAttributeView;
+import java.nio.file.attribute.UserPrincipal;
+
 /**
  * Unit tests
  */
 public class ScannerTest 
 {
     @Test
-    public void scanLocalImageTest()
-    {
-        String license = "r+wzvemF4mHMIBkvKHXTuuLzw4ML/RW2ARfbC06Qem1VdQ8DXcMWNKbd8GD0uYiLoS9d9XPM8bff4ag66fqwdOa42oNXk9dzP1TTRg0zvVKJNMxbxKAuGMY24x0NlXstc3Jl2BcXq7aTHrlQwzJ/7qD8157+YfcjKJbywbt3WvvdpuoJtJIuKaO9hOfGu9/zbRZznEtZiJj56xaRMVZ/ZkPHIMseRWML7FUr1oft20PYU7ltf1BGPT/LlocAXAJucBpUz5iosRzS1iYJhU4M1kdBhEwQHl6FZXrtzJGYpM21587FyYrkvoIPQNlIijaAOiCXR060PJNfcQzOzEpYRidhVPnHIHQuIn8gcYuU3XwPB9Pq3tGzERpr0aB64EjSSanpcIh/XEhnXijiRP6CfxsHx8RvD/jkXy5VlJpTVFh1Iu7ywE6fePHb3hIe16uCtdYB8fu2p8ksCcnxXbjBUkKvdLLxq5gvOCfCJxknd4GdIS3UJySNblgmlfU6liYcgt555TitN5DAjuMhTaoxj9FoPPwJsl4rrf2+32cqUFI+9+UdNsSd2gWtS/8T2ifPHNVMA7fdPdIWx+/4jXP8u484cLqGG9RutNGPo0k7cmxmgx29FvVOMvpkotD8IZsbCfl8sFHtFRP7PegWYPghZf6yUkw5m4SD+bacVZalnLizZKbDgFuzPmH1LgrroVN1Ngf06JVfO6lDodbAOactsJC7P8DOPRUoOtnCf3ffgshPh+QSTKDOmNvFU51VB7MoLlZiKNaVP6yMqS1bUwxRpI5eehqhqCHy46PWfRHF2EnbiQOyYIjJSY5jE54TzOaKZ613GgdIaMN+mbXg+UWajDevwCvKQo+CrwpBoX72tP/as9G3zIJILaKuXbQRvTtZbgGREMSoLIDxPBYe3FVqMHFg/jUE9iPbsF16JHv/z3EzcAYKwYSyDEyVppTvLjQY0t/Vul4SEj+QWpyW7elAUmatzHdAcX2SB3oDOhUtDyIxw8rtenD9hCOfspJtEFknhtM+Zm1WuoQYRM5/paEVvbHZnu2XHY2cnBtxJZAD8wKktnSPGg3foCXQpQcSEyhfWhVT14WX2/GxgbdwIuxSBcoHlcduca5tKtlKAhJ4kjPPZeQc5A18z48pLRU=";
+    public void scanLocalImageTest() throws IOException {
+        String license = "";
 
-        String imageName = "songlongtj/alpine";
+        String imageName = "alpine";
         String imageTag = "3.6";
         String nvScannerImage = "neuvector/scanner:latest";
-        String nvRegistryUser = "xxx";
-        String nvRegistryPassword = "xxx";
+        String nvRegistryUser = "";
+        String nvRegistryPassword = "";
         String nvRegistryURL = "https://registry.hub.docker.com";
         String mountPath = "/temp";  //mountPath is an optional parameter. It will use "/var/neuvector" by default.
 
         Image image = new Image(imageName, imageTag);
         NVScanner scanner = new NVScanner(nvScannerImage, nvRegistryURL, nvRegistryUser, nvRegistryPassword, mountPath);
 
         ScanRepoReportData scanReportData = Scanner.scanLocalImage(image,scanner,license);
+
+        File file = new File(mountPath + "/scan_result.json");
+        UserPrincipal user = getUserPrincipal(mountPath, file);
+
+        assertTrue(user.getName().equals("root") || user.getName().contains(System.getProperty("user.name")));
```

Comment:
After your changes `assertTrue(user.getName().equals("root")` should never be true.. So we should be asserting the opposite..  Currently this assertion is not testing anything because user principal will either be root or the user anyway.

Reply:
a6e52d49e0973fb081e8cbe3ee4cc56a3287e22b